### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by manujagobind**
**Summary**
- Add product listing page
- Add product filter
- Update index page with newly added pages

**Live Preview**
[Click Here to View on Netlify](https://deploy-preview-3--commerce-one.netlify.app/src/pages/products.html)

### CODE REVIEW

File: src/pages/products.html [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+              <input type="radio" name="rating" value="rating-filter1" id="rating-filter1" />
+              <label for="rating-filter1">4 and above</label>
+            </div>
+            <div>
+              <input type="radio" name="rating" value="rating-filter2" id="rating-filter2" />
+              <label for="rating-filter2">3 and above</label>
+            </div>
+            <div>
+              <input type="radio" name="rating" value="rating-filter3" id="rating-filter3" />
+              <label for="rating-filter2">2 and above</label>
```
Comment (darshandamre) [inline src/pages/products.html]: correct this `for` value

**Review by darshandamre — COMMENTED** [review]
looks good just make one correction

File: src/pages/products.html (hunk 1 above)
Comment (pariyar07) [inline src/pages/products.html]: label 'for doesn't match

**Review by pariyar07 — APPROVED** [review]
Looks good to me please go ahead.

**Pull request merged**